features/messages/impl/src/main/res/values/temporary.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <string name="crypto_history_visible">Messages you send will be shared with new members invited to this room. %1$s</string>
```

Comment:
Do you need any help to add this string to Localazy? I understand that this is temporary, but It can't stay here, it won't be translated in other languages.